libyul/AST.h

```diff
@@ -41,7 +43,41 @@ using TypedNameList = std::vector<TypedName>;
 
 /// Literal number or string (up to 32 bytes)
 enum class LiteralKind { Number, Boolean, String };
-struct Literal { langutil::DebugData::ConstPtr debugData; LiteralKind kind; YulString value; Type type; };
+/// Literal value that holds a u256 word of data, can be of LiteralKind type and - in case of arguments to
+/// builtins - exceed the u256 word (32 characters), in which case the value is set to `unlimited`
+class LiteralValue {
```

Comment:
```suggestion
/// builtins - exceed the u256 word (32 bytes), in which case the value is set to `unlimited`
class LiteralValue
{
```

In hex that would actually be 64 characters :)